src/02-map.js

```diff
@@ -35,7 +39,9 @@ function getSongDetails(songs) {}
  *  getTitleAndArtist(songs);
  *  //> [ { "Berlin Tsukin": "Taiyo Ky" }, { Up: "Sebastian Kamae" }, ... ]
  */
-function getTitleAndArtist(songs) {}
+function getTitleAndArtist(songs) {
+  songs.map(song => `${song.title} : ${song.artist}`)
```

Comment:
your code here is returning a string but what you want to return is an object. What is the correct object syntax?
it should look something like {`${song.title}` : `${song.artist}`}